index.js

```diff
@@ -21,7 +23,55 @@ mongoose
   })
   .then(() => {
     // Run your code here, after you have insured that the connection was made
+   /* THIS IS ITERATION 2!!!!!!!!!
+    Recipe.create( data[0] )
+      .then((recipe) => {
+        console.log(recipe.title)
+
+      })
+      .catch(()=> {
+        console.log()
+      })
+      */
+
+     Recipe.insertMany(data)
+        .then((recipes) => {
+          //console.log(recipes)
+          for(let recipe in recipes){
+            console.log(recipes[recipe].title)
+          }
+
+          let updatedPromise = Recipe.updateOne({title: "Rigatoni alla Genovese"}, {$set: {duration: 100}})
+
+            updatedPromise.then(() => {
+                console.log('Updated yo!')
```

Comment:
since you are already using promise.all later you don't need to check if the promise resolved here as well. Instead you can put this console.log inside the .then of promise.all